Engineering-Reference/GoogleDestinationAO.md

```diff
@@ -43,71 +43,71 @@ Default Value: Off
 
 Migrated Inbox email directly into 'All Mail' within Google Workspace and not to the Inbox.
 
-Change Conditions: ??
+Change Conditions: Enabling will remove the Inbox label and effectively archive inbox email.
```

Comment:
Don't think this is correct.   "All Mail" Is a folder inbox would be a sub folder. this would just remove the sub folder and put all the mail into the "All Mail" folder.   If you looking at gmail this would take everything and move it to the Everything else section and not what is important to you

Reply:
Its labels not folders, in Gmail, when you 'Archive' an email it simply strips the labels, thus appearing only in All Mail - which is just a view of everything. So this is correct, but I will make it clearer.